packages/video_player_avplay/tizen/src/video_player.cc

```diff
@@ -115,6 +117,7 @@ void VideoPlayer::SendInitialized() {
          flutter::EncodableValue(duration_range)},
         {flutter::EncodableValue("width"), flutter::EncodableValue(width)},
         {flutter::EncodableValue("height"), flutter::EncodableValue(height)},
+        {flutter::EncodableValue("isLive"), flutter::EncodableValue(is_live)},
```

Comment:
Actually, we can change to  {flutter::EncodableValue("isLive"), flutter::EncodableValue(IsLive())}